subtrack.MAUI/Utilities/CssUtil.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace subtrack.MAUI.Utilities;
+
+public static class CssUtil
+{
+    public static string GetDueClass(int daysUntilDue) =>
+        daysUntilDue switch
+        {
+            <= 0 => "overdue",
```

Comment:
only less than 0 is overdue, 0 or 2 is warning. 0 = should be paid today

Reply:
Resolved in latest commit.